src/extract.js

```diff
@@ -0,0 +1,25 @@
+import { fromCamelToSentence } from './utils';
+
+const extract = (props = {}) => {
```

Comment:
A comment to this function might help to understand what are we extracting here.